staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -244,6 +244,13 @@ message Mount {
     // - when set to true, readonly must be explicitly set to true, and propagation must be PRIVATE (0).
     // - (readonly == false && recursive_read_only == false) does not make the mount read-only.
     bool recursive_read_only = 8;
+    // Mount an image reference (image ID or digest), which is a special use
```

Comment:
```suggestion
    // Mount an image reference (image ID, with or without digest), which is a special use
```

Reply:
(I assume that a raw digest without an image ref is not supported)

Reply:
Fixed as suggested :+1: